config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+/Iz4Ghnj7SukvCX3yYNcS4AkvkCxzdYfV4RWvjsbZhAGXlERaMZutWYJPpwUvf+9KFFkG4at8N9L9givtwOd2NAT3K2ozJw8byBsVUx8ccxGTNJUZVWFQ1OWdH+TnksDne2OGG/1W6yPm/fS4TyPknLO1piC4eA/g3bz+jOJDRz/DB6NwqIv9C3dxU4fCL9wpQif1CjxtPxJEPxGimMFle9Bkr9nIzBY8lnQPKvpan3JAtu4tu9mXbbzWvpbJXutDB+J9wUCTvqr5Rs6xQvBlrVk/jsPB8fDj7kLHtAmhe6jWkNTlv2pn39lWFoF3JzJyP+o6rj8h98tRyaly86XwrwAYlqLStrvqSIjc6IKRDZJypLZ+QdK6Q+tNX+2InfqfCmugjsmKyuIYiLRu3H6rkeNkb2B1+/JI+O/--wd5VguwJmv5qz5hj--TdmQof30/vIptQwPb4Ja8w==
```

Comment:
As these are encrypted secrets, might it be a potential security vulnerability to track these within the public repository?